.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+## Status
+**READY/IN DEVELOPMENT/HOLD**
+
+#### What does this PR do?
+A few sentences describing the overall goals of the pull request's commits.
+Why are we making these changes? Is there more work to be done to fully
+achieve these goals?
+
+#### Helpful background context (if appropriate)
+
+#### How can a reviewer manually see the effects of these changes?
+
+#### What are the relevant tickets?
+- https://mitlibraries.atlassian.net/browse/DISCO-
+
+#### Screenshots (if appropriate)
+
+#### Todo:
+- [ ] Tests
+- [ ] Documentation
+- [ ] Stakeholder approval
+
+#### Requires Database Migrations?
```

Comment:
I just noticed this, which is probably not required for this project.

Reply:
Yeah I think you are correct